src/bosh-google-cpi/google/config/config.go

```diff
@@ -4,13 +4,27 @@ import (
 	bosherr "github.com/cloudfoundry/bosh-utils/errors"
 )
 
+var CpiRelease string
```

Comment:
Is there any need to access this outside of config other than for testing? 

If not: You can instead create a new test file and put it in the `config` package (instead of `config_test`) and access/mutate private data. A common convention for this is `config_internal_test.go`.